src/box/port.c

```diff
@@ -295,7 +295,7 @@ port_c_get_msgpack(struct port *base, uint32_t *size)
 extern void
 port_c_dump_lua(struct port *port, struct lua_State *L, bool is_flat);
 
-extern struct sql_value *
+extern struct Mem *
```

Comment:
I would suggest doing it in a different patch-set. `struct Mem` was added at the same time sqlite sources were added: https://github.com/tarantool/tarantool/blob/d0a9dfed5e31213aec5859015d9869e235ba6d44/src/box/sql/vdbeInt.h#L187

Please fill an issue, I will do it later.

Reply:
Using the name `Mem` in the core sources is confusing, because it's unclear what this "Mem" relates to. Besides, this name doesn't meet  the Tarantool naming convention. Judging by the comment to `struct Mem`, I think that this type should in fact be called `sql_value`. I see no reason in renaming `sql_value` to `Mem` in the core sources just to rename it back later. I think it's better to leave the typedef in place for now to be used outside `src/box/sql`.